connector/docker-integration-tests/src/test/scala/org/apache/spark/sql/jdbc/v2/DockerJDBCIntegrationV2Suite.scala

```diff
@@ -57,7 +57,11 @@ abstract class DockerJDBCIntegrationV2Suite extends DockerJDBCIntegrationSuite {
     connection.prepareStatement("INSERT INTO pattern_testing_table "
         + "VALUES ('special_character_underscorenot_present')")
       .executeUpdate()
+
+    insertDatetimeData(connection)
```

Comment:
is there a standard INSERT command that works in all databases for datetime?

Reply:
It's very hard. Each database integration test needs to use its own method.